a3p-integration/proposals/z:acceptance/psm.test.js

```diff
@@ -0,0 +1,267 @@
+/**
+ * @file The goal of this file is to implement a set of tests to make sure PSM works properly
+ *
+ * Here are the steps we want to take;
+ * 1 - Change swap fees and mint limit according to "psmTestSpecs" below
+ * 2 - Create a new user using agd.keys
+ * 3 - Fund new user with a stable coin from the VALIDATOR
+ *     - Do not provision manually
+ * 4 - Make sure new user is able to mint IST from PSM (fees are applied)
+ * 5 - Make sure new user can pay their debt and get their anchor (fees are applied)
+ * 6 - Make sure mint limit is adhered
+ */
+
+import test from 'ava';
+import {
+  adjustBalancesIfNotProvisioned,
+  agopsPsm,
+  bankSend,
+  checkGovParams,
+  checkSwapExceedMintLimit,
+  checkSwapSucceeded,
+  checkUserInitializedSuccessfully,
+  getPsmMetrics,
+  implementPsmGovParamChange,
+  initializeNewUser,
+  maxMintBelowLimit,
+} from './test-lib/psm-lib.js';
+import {
+  getUser,
+  GOV1ADDR,
+  GOV2ADDR,
+  GOV3ADDR,
+  waitForBlock,
+} from '@agoric/synthetic-chain';
+import { getBalances } from './test-lib/utils.js';
+
+// Export these from synthetic-chain?
```

Comment:
perhaps eventually. But having them in ENV gives us flexibility for other runtimes using them

Reply:
Why not we do them both? We do the same for `ATOM_DENOM`  => 
https://github.com/Agoric/agoric-3-proposals/blob/93bb953db209433499db08ae563942d1bf7eeb46/packages/synthetic-chain/src/lib/constants.js#L11

Reply:
sure. Just if they're in ENV that's the source of truth and any module can be made to export them